charts/datahub/subcharts/acryl-datahub-actions/templates/deployment.yaml

```diff
@@ -159,6 +164,10 @@ spec:
                   name: {{ .Values.global.datahub.metadata_service_authentication.systemClientSecret.secretRef }}
                   key: {{ .Values.global.datahub.metadata_service_authentication.systemClientSecret.secretKey }}
             {{- end }}
+            {{- if or .Values.global.datahub.monitoring.enablePrometheus }}
```

Comment:
```suggestion
            {{- if .Values.global.datahub.monitoring.enablePrometheus }}
```

Reply:
Woops, that was from copy-pasting from one of the other deployments...there is another (not really used anymore?) option to enable monitoring "enableJMXPort", which I have not added to the datahub-actions chart: https://github.com/acryldata/datahub-helm/blob/f79d747bd186c82ff3734a32a4bb68923db8bc9b/charts/datahub/subcharts/datahub-frontend/templates/deployment.yaml#L83C10-L83C38

Reply:
Good point!

The `enableJMXPort` was introduced here in PR #110 
Since missing value defaults to `false`, the actual `if` evaluation fully depends on `enablePrometheus` flag.

IMO and just for consistency with the existing code, you may add it to the `if` statement as well
https://github.com/search?q=repo%3Aacryldata%2Fdatahub-helm+enableJMXPort&type=code

Reply:
Not sure whether wie should also add enableJMXPort here...although I am not sure about the Java-Containers and if this is really a JMX port (in fact it is only a simple HTTP request which Prometheus is performing for the "JMX" port?), but in case of datahub-actions it is definitely not a JMX port and only a Prometheus endpoint: https://github.com/acryldata/datahub-actions/blob/2f78ab1a27f795db8541d1d6a783a08ad460dca5/datahub-actions/src/datahub_actions/entrypoints.py#L22

Reply:
indeed

Reply:
let's keep it as it is then